source/documentation/deploying_services/use_a_custom_domain.md

```diff
@@ -105,7 +105,15 @@ After the record is created wait for up to an hour for the CloudFront distributi
 
 #### CDN Configuration Options
 
+##### Options available
+
+Name | Required | Description | Default
+--- | --- | --- | ---
+`domain` | *Required* | Your custom domain (or domains separated by commas) |
+`cookies` | *Optional* | Forward cookies to the origin | `true` |
+`headers` | *Optional* | A list of headers to forward to the origin | `["Host"]` |
 
```

Comment:
... and then a section here explaining how tenants should use the table to choose the right option for their needs. We should explicitly state here that more info on each can be found in the relevant sections below.